message PBVortexLine {
  repeated double vertices = 1 [ packed = true ];
  optional uint32 id = 2;
  optional bool bezier = 4 [ default = false ];
  optional bool loop = 5 [ default = false ];
  
  optional uint32 timestep = 3;
  optional double time = 6;
};

message PBVortexLines {
  repeated PBVortexLine lines = 1;
  // optional PBDataInfo info = 2;
  optional bytes info_bytes = 2;
};
